python/tchannel/tornado/message_factory.py

```diff
@@ -0,0 +1,98 @@
+from ..messages import Types
+from ..exceptions import StreamingException
+from ..messages.common import FlagsType
+from ..messages.call_continue import CallContinueMessage
+
+
+class MessageFactory(object):
+
+    # 64KB Max frame size
+    # 16B (size:2 | type:1 | reserved:1 | id:4 | reserved:8)
+    # 1 2 Bytes can represent 0~64KB-1
+    MAX_PAYLOAD_SIZE = 0xFFEF   # 64*1024 - 16 - 1
+
+    def __init__(self):
+        """key: message_id
+           value: incomplete messages
```

Comment:
This shouldn't be a docstring on the `__init__` method but a regular `#` comment on `message_buffer`.